### PR TITLE
use title in title html tag

### LINKED ISSUE #42: Title tag is always Coopdevs
<Title> is static, always "Coopdevs" 

title should change dynamically, in posts is the title of post

### PR DESCRIPTION
**Pull request by sseerrggii**
Fix #42 but I noticed that title used in Jobs are like variables and I don't know how to show it properly....

### CODE REVIEW

**Comment by sseerrggii** [comment]
@sauloperez can you review this?

**Review by enricostano — APPROVED** [review]

File: _includes/head.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>Coopdevs</title>
+  <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
```
Comment (naoisegolden) [inline _includes/head.html]: Where is `site.title` defined?

In any case I would recommend doing something like this: 

```suggestion
  <title>{{ site.title }} - {% if page.title %}{{ page.title }}{% endif %}</title>
```

(and defining `site.title` if it's not defined). Careful, this change would render "Coopdevs - Coopdevs" in the home page.

**Comment by sseerrggii** [comment]
Merge?

**Comment by enricostano** [comment]
@sseerrggii `site.title` is defined or not? I still don't understand.

**Comment by sseerrggii** [comment]
@enricostano site.title wasn't defined, fixed! And improve some titles, the problem is in titles that are defined as vars for translate, how can we show it? 

Peek 09-05-2019 13-27

**Review by enricostano — APPROVED** [review]

**Pull request merged**